casper/src/main/scala/coop/rchain/casper/Casper.scala

```diff
@@ -285,6 +268,33 @@ sealed abstract class MultiParentCasperInstances {
           _      <- addEffects(status, b)
         } yield status
 
+      private def validateTransactions(
```

Comment:
I think this function probably belongs in `Validate.scala` and should just be named `transactions` there. Then in casper we call `Validate.transactions(...)`, or maybe it could even be included in the `validateBlockSummary` function